tests/acceptance/tests/Settings/FlowBuilderTemplates.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { expect, test } from '@fixtures/AcceptanceTest';
+import { getFlowId, compareFlowTemplateWithFlow } from "@shopware-ag/acceptance-test-suite";
+
+test('As an admin, I want to create new flows from templates, so that I can easily create new ones based on the default flows.', { tag: '@Flow' }, async ({
+        ShopAdmin,
+        AdminFlowBuilderTemplates,
+        AdminFlowBuilderCreate,
+        AdminFlowBuilderDetail,
+        IdProvider,
+        AdminApiContext,
+
+    }) => {
+
+    // change flowTemplateName to test different flows/templates
```

Comment:
these commented lines are good reason that we should use test.step to better describe test case

```suggestion
    await test.step('Change flowTemplateName to test different flows/templates', async () => 
```

Reply:
Thanks, @vanpham-sw; I split the test into two steps. I am often uncertain about when to split. Are there any guidelines on that?

Reply:
We don't have any guidelines yet but I'm working on one, for now I can suggest:
- Descriptive Names: Use clear and descriptive names for each step. Each step's name should explain what the step is doing or what it's verifying.
- Logical Grouping: Group related actions and assertions into a single step. This could be filling out a form, or verifying a set of conditions.
- Keep Steps Focused: If a step is doing too much, consider breaking it into smaller steps.

Will keep you posted